### ISSUE TITLE
Unable to run Python script with dependencies using RealTimeSync

### ISSUE THREAD

**Issue by randoogle** [flathub/org.freefilesync.FreeFileSync#79]
I have my own custom python script that I want to run every time a folder's content changes.
In nutshell, it's a folder full of the latest photos from my phone. When a new photo gets added my script copies it into another directory structure that I use to keep my photos organized by date.
Anyway, my python script has dependencies which I've installed on my OS python. Running it from outside the flatpak it runs fine, but I find that when running from within the flatpak, RealTimeSync want's to use it's own internal version of python, which does not include the dependencies I need, and so it fails.

Is there a way to indicate that I want to run an external binary from the RealTimeSync command line, or from the flatpak cli?

**Comment by randoogle** [comment]
Okay I tried a bunch of different things, but I finally found **flatpak-spawn**
I also had to enable "D-Buss session bus" using Flatseal (I assume there's some possible security concerns to doing this)

Anyway, for anyone else running into this, simply add "flatpak-spawn --host" in front of your call to python. Example:
`flatpak-spawn --host python3 moveFiles.py --source /source/dir --dest /dest/dir`

Voila. Now it will use your host's instance of python with all installed dependencies.
